app/components/record_actions/view.rb

```diff
@@ -21,7 +21,13 @@ def can_recommend_for_award?
     def action_links
       return "#{reinstate_link} or #{withdraw_link}".html_safe if trainee.deferred?
 
-      "#{defer_link} or #{withdraw_link}".html_safe
+      if trainee_commencement_date_in_future?
```

Comment:
I'm wondering whether we need to push these methods down the model as they'll need to be used in other areas. Eg I need to use similar logic in my branch

Reply:
We could certainly do something share the behaviour. I'll inclined to keep it here until we're ready to pull it out.